app/league/slack_messenger.py

```diff
@@ -28,6 +26,9 @@ def init_app(self, app):
 
     def update_configuration(self, config):
         """Update Slack Messenger configuration."""
+        assert 'enabled' in config
```

Comment:
What I meant was, if it violated the condition, it would raise a runtime exception. Which can be fine. I just wanna be clear about under what circumstances this would occur. Currently I am unclear on this. Can you explain?